src/Core/Types/LegacyIAccessibleRoleType.cs

```diff
@@ -0,0 +1,120 @@
+        public const int ROLE_SYSTEM_TABLE = 24;
+        public const int ROLE_SYSTEM_COLUMNHEADER = 25;
+        public const int ROLE_SYSTEM_ROWHEADER = 26;
+        public const int ROLE_SYSTEM_COLUMN = 27;
+        public const int ROLE_SYSTEM_ROW = 28;
+        public const int ROLE_SYSTEM_CELL = 29;
+        public const int ROLE_SYSTEM_LINK = 30;
+        public const int ROLE_SYSTEM_HELPBALLOON = 31;
+        public const int ROLE_SYSTEM_CHARACTER = 32;
+        public const int ROLE_SYSTEM_LIST = 33;
+        public const int ROLE_SYSTEM_LISTITEM = 34;
+        public const int ROLE_SYSTEM_OUTLINE = 35;
+        public const int ROLE_SYSTEM_OUTLINEITEM = 36;
+        public const int ROLE_SYSTEM_PAGETAB = 37;
+        public const int ROLE_SYSTEM_PROPERTYPAGE = 38;
+        public const int ROLE_SYSTEM_INDICATOR = 39;
+        public const int ROLE_SYSTEM_GRAPHIC = 40;
+        public const int ROLE_SYSTEM_STATICTEXT = 41;
+        public const int ROLE_SYSTEM_TEXT = 42;
+        public const int ROLE_SYSTEM_PUSHBUTTON = 43;
+        public const int ROLE_SYSTEM_CHECKBUTTON = 44;
+        public const int ROLE_SYSTEM_RADIOBUTTON = 45;
+        public const int ROLE_SYSTEM_COMBOBOX = 46;
+        public const int ROLE_SYSTEM_DROPLIST = 47;
+        public const int ROLE_SYSTEM_PROGRESSBAR = 48;
+        public const int ROLE_SYSTEM_DIAL = 49;
+        public const int ROLE_SYSTEM_HOTKEYFIELD = 50;
+        public const int ROLE_SYSTEM_SLIDER = 51;
+        public const int ROLE_SYSTEM_SPINBUTTON = 52;
+        public const int ROLE_SYSTEM_DIAGRAM = 53;
+        public const int ROLE_SYSTEM_ANIMATION = 54;
+        public const int ROLE_SYSTEM_EQUATION = 55;
+        public const int ROLE_SYSTEM_BUTTONDROPDOWN = 56;
+        public const int ROLE_SYSTEM_BUTTONMENU = 57;
+        public const int ROLE_SYSTEM_BUTTONDROPDOWNGRID = 58;
+        public const int ROLE_SYSTEM_WHITESPACE = 59;
+        public const int ROLE_SYSTEM_PAGETABLIST = 60;
+        public const int ROLE_SYSTEM_CLOCK = 61;
+        public const int ROLE_SYSTEM_SPLITBUTTON = 62;
+        public const int ROLE_SYSTEM_IPADDRESS = 63;
+        public const int ROLE_SYSTEM_OUTLINEBUTTON = 64;
+#pragma warning restore CA1707 // Identifiers should not contain underscores
+
+        private static LegacyIAccessibleRoleType TheInstance;
+
+#pragma warning disable CA1024 // Use properties where appropriate
+        /// <summary>
+        /// static method to get an instance of this class
+        /// singleton
+        /// </summary>
+        /// <returns></returns>
+        public static LegacyIAccessibleRoleType GetInstance()
+        {
+            if (TheInstance == null)
+            {
+                TheInstance = new LegacyIAccessibleRoleType();
+            }
+
+            return TheInstance;
+        }
+#pragma warning restore CA1024 // Use properties where appropriate
+
+        /// <summary>
+        /// private constructor since it would be singleton model
```

Comment:
nit: can we make this comment match the changes you made in https://github.com/microsoft/axe-windows/pull/823?